### PR TITLE
Fix(#941): Replacing nested resolvers with inline requests to reduce response time.

### PR DESCRIPTION
**Pull request by ghanlohar**
# Fixes - ONEPLAT-941

# Explain the feature/fix

Replacing nested resolvers with inline requests to reduce listVideos response time.

## Does this PR introduce a breaking change
No

### CODE REVIEW

File: video-library-service/service.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ const apollo = new ApolloServer({
     stack: error.stack ? error.stack.split('\n') : [],
     path: error.path,
   }),
-  extensions
+  //extensions
```
Comment (riginoommen) [inline video-library-service/service.ts:59]:
```suggestion
  extensions
```

File: video-library-service/src/resolver.ts [hunk 2]
```diff
@@ -52,23 +56,32 @@ export const VideoLibraryResolver = {
...
               text: `Hello,
 
-${video.createdBy} has recently added a video to the One Portal Video Library.
+${userDetails?.name} has recently added a video to the One Portal Video Library.
 
 Video Title: ${video.title}
 
 Video Description:
-\t${video.description || `A Demo was recently created for ${video.title} by ${video.createdBy}.`}
+\t${video.description || `A Demo was recently created for ${video.title} by ${userDetails?.name}.`}
```
Comment (riginoommen) [inline video-library-service/src/resolver.ts:84]:
```suggestion
\t${video.description || `A demo was recently created for ${video.title} by ${userDetails?.name}.`}
```

File: video-library-service/src/resolver.ts [hunk 3]
```diff
@@ -52,23 +56,32 @@ export const VideoLibraryResolver = {
...
+        .then(async video => {
+          const query = `{
+            getUsersBy(rhatUUID: "${video.createdBy}") {
+              name
+              rhatUUID
+            }
+          }`;
+          const userDetails = await VideoLibraryHelper.fetchUserDetails(query)
+          .then( res => res.data?.getUsersBy ? res.data.getUsersBy[0] : null );
+          console.log("User Details", userDetails);
```
Comment (riginoommen) [inline video-library-service/src/resolver.ts:68]:
```suggestion
          
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: video-library-service/service.ts (hunk 1 above)
Comment (ghanlohar) [reply video-library-service/service.ts]: Done!

File: video-library-service/src/resolver.ts (hunk 2 above)
Comment (ghanlohar) [reply video-library-service/src/resolver.ts]: Done!

**Review by riginoommen — APPROVED** [review]

**Pull request merged**